backend/routes/graph.py

```diff
@@ -42,84 +36,98 @@
 
 
 @router.get("/concept-graph")
-async def concept_graph(request: Request, codeset_ids: List[int] = Query(...),
-                        extra_concept_ids: Optional[List[int]] = Query(None)) -> List:
+async def concept_graph_get(
+    request: Request, codeset_ids: List[int] = Query(...), extra_concept_ids: Optional[List[int]] = Query(None),
+    hide_vocabs = ['RxNorm Extension'], verbose = VERBOSE
+) -> Dict:
+    """Return concept graph"""
     extra_concept_ids = extra_concept_ids if extra_concept_ids else []
-    return await concept_graph_post(
-        request=request, codeset_ids=codeset_ids, extra_concept_ids=extra_concept_ids)
+    return await concept_graph_post(request, codeset_ids, extra_concept_ids, hide_vocabs, verbose)
 
 
-# TODO: Joe refactor: after get_concepts_for_graph()
-#  - if look at graph-testing.md, there are some codeset_ids i can use
+# TODO
 @router.post("/concept-graph")
 async def concept_graph_post(
     request: Request, codeset_ids: List[int], extra_concept_ids: Union[List[int], None] = [],
-    hide_vocabs = ['RxNorm Extension'], verbose = False
-) -> List:
+    hide_vocabs = ['RxNorm Extension'], verbose = VERBOSE
```

Comment:
## `concept_graph()`: Minor updates
- Created non-route `concept_graph()`
  - This was necessary to make unit test; I couldn't pass `Request` obj
- `concept_graph_post()` is now a fairly dumb route
- Codestyle fixes
- Connected unused params from `concept_graph_get()` to `concept_graph_post()`
- `verbose=False` -> `VERBOSE` (`True`)
  - Should I change this back?
- `extra_concept_ids`: Added a todo. Passing it now throws a `NotImplementedError`